apps/gnome-mobile/src/app/(app)/achievements.tsx

```diff
@@ -52,28 +54,23 @@ export default function AchievementScreen() {
           numColumns={3}
           renderItem={({ item }) => (
             <Achievement
-              onPress={() => setSelectedAchievementId(item.id)}
+              onPress={() => {
+                setSelectedAchievementId(item.id);
+                achivementRef.current?.present();
+              }}
               title={item.name}
             />
           )}
           keyExtractor={(item) => item.id}
         />
       </View>
 
-      {selectedAchievementId !== undefined && (
-        <BottomSheet
-          handleIndicatorClassName={"bg-tekst w-24 mt-2 rounded-lg"}
-          backgroundClassName={"bg-background"}
-          enableDismissOnClose
-          enablePanDownToClose
-          index={0}
-          onClose={() => setSelectedAchievementId(undefined)}
-        >
-          <AchievementDetailsBottomSheet
-            achievementId={selectedAchievementId}
-          />
-        </BottomSheet>
-      )}
+      <BottomSheetWrapper
+        ref={achivementRef}
+        onDismiss={() => setSelectedAchievementId(undefined)}
```

Comment:
skoro to jest ref to nie trzeba jeszcze w ondismissie wywołać zamknięcia? czy ponowne otwarcie tego samego lub innego sheeta dalej będzie śmigać?